nxt/test/test_cli.py

```diff
@@ -8,23 +10,63 @@
 from nxt.session import Session
 from nxt.test import get_test_file_path
 
-GRAPH_PATH = get_test_file_path("StageRuntimeScope.nxt")
 
 class CLI(unittest.TestCase):
+    '''The test graph writes /cli_test.my_attr to the location specified by
+    The env var $_TEST_WRITE_PATH, allowing verification of paramter overrides
+    by reading the resulting file.'''
+    test_graph = get_test_file_path('cli_test.nxt')
+
+    def setUp(self):
+        self.test_path = get_test_file_path('delete_me')
+        # Test graph writes a file to the location of this env var.
+        self.test_env = {'_TEST_WRITE_PATH': self.test_path}
+
+    def tearDown(self):
+        try:
+            os.remove(self.test_path)
+        except FileNotFoundError:
+            pass
+
+    def read_test_path(self):
+        with open(self.test_path, 'r') as fp:
+            contents = fp.read()
+        return contents
```

Comment:
Recently learned you can _just_ do this:
```suggestion
        with open(self.test_path, 'r') as fp:
            return fp.read()
```